setup.py

```diff
@@ -86,7 +86,10 @@ def get_reqs(lookup=None, key="INSTALL_REQUIRES"):
         long_description_content_type="text/markdown",
         keywords=KEYWORDS,
         setup_requires=["pytest-runner"],
-        install_requires=INSTALL_REQUIRES,
+        install_requires=INSTALL_REQUIRES
+        + [
+            "fake-useragent @ git+https://github.com/danger89/fake-useragent@master#egg=fake-useragent"
```

Comment:
cant we just use pip instead of a link to gh here? or why are we using github instead of pip? is the dev version better than the released?

Reply:
Since the setup.py is controlling how pip installs the entire package, it needs to be in this format to work. Otherwise the installing user would have to run two separate commands, which is weird. You can try it out locally to see how it works!